### PR TITLE
New term entry - d3 .scaleOrdinal()

### LINKED ISSUE #3885: [Term Entry] D3 Scaling .scaleOrdinal()
### Reason/inspiration (optional)

We would like a new term entry on the `.scaleOrdinal()` method for [D3](https://www.codecademy.com/resources/docs/d3): Scaling. The entry should go in a new file under `docs/content/d3/concepts/scaling/terms/scaleOrdinal/scaleOrdinal.md`.

The entry should describe what the method does. Be sure to include the following sections:

- A "Syntax" section with a ` ```pseudo` code snippet.
- An "Example" section with a ` ```js `code snippet.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by A-Kiwams**
### Description

Closes #3885 

New entry added: d3 .scalOrdinal()

### Type of Change

- Adding a new entry

### Checklist

- [x ] All writings are my own.
- [x ] My entry follows the Codecademy Docs style guide.
- [x ] My changes generate no new warnings.
- [x ] I have performed a self-review of my own writing and code.
- [x ] I have checked my entry and corrected any misspellings.
- [x ] I have made corresponding changes to the documentation if needed.
- [x ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by dakshdeepHERE** [comment]
I'm marking this invalid till you fix the issues in this entry @A-Kiwams.

**Comment by A-Kiwams** [comment]
Looking into it

File: content/d3/concepts/scaling/terms/domain/scaleOrdinal/scaleOrdinal.md
Comment (dakshdeepHERE) [inline content/d3/concepts/scaling/terms/domain/scaleOrdinal/scaleOrdinal.md:3]: make this description a small explanation of what scaleOrdinal() does and then explain it more below this metadata.

File: content/d3/concepts/scaling/terms/domain/scaleOrdinal/scaleOrdinal.md
Comment (dakshdeepHERE) [inline content/d3/concepts/scaling/terms/domain/scaleOrdinal/scaleOrdinal.md:27]: This sentence should tell us what the `.scaleOrdinal()` is doing in the example below.

File: content/d3/concepts/scaling/terms/domain/scaleOrdinal/scaleOrdinal.md
Comment (dakshdeepHERE) [inline content/d3/concepts/scaling/terms/domain/scaleOrdinal/scaleOrdinal.md:33]: remove this and add the shell block. Like you insert the code blocks in `pseudo` and `js` tags. Use the shell tag and then put the output of the code in it.

File: content/d3/concepts/scaling/terms/domain/scaleOrdinal/scaleOrdinal.md
Comment (dakshdeepHERE) [inline content/d3/concepts/scaling/terms/domain/scaleOrdinal/scaleOrdinal.md:42]: what is this? The output for the above code?

File: content/d3/concepts/scaling/terms/domain/scaleOrdinal/scaleOrdinal.md
Comment (dakshdeepHERE) [inline content/d3/concepts/scaling/terms/domain/scaleOrdinal/scaleOrdinal.md:49]:
```suggestion
```

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
More comments for you to work on @A-Kiwams.

File: content/d3/concepts/scaling/terms/domain/scaleOrdinal/scaleOrdinal.md
Comment (A-Kiwams) [reply content/d3/concepts/scaling/terms/domain/scaleOrdinal/scaleOrdinal.md:3]: Alright

File: content/d3/concepts/scaling/terms/domain/scaleOrdinal/scaleOrdinal.md
Comment (A-Kiwams) [reply content/d3/concepts/scaling/terms/domain/scaleOrdinal/scaleOrdinal.md:33]: Noted

File: content/d3/concepts/scaling/terms/domain/scaleOrdinal/scaleOrdinal.md
Comment (A-Kiwams) [reply content/d3/concepts/scaling/terms/domain/scaleOrdinal/scaleOrdinal.md:42]: Output of the code

File: content/d3/concepts/scaling/terms/domain/scaleOrdinal/scaleOrdinal.md
Comment (A-Kiwams) [reply content/d3/concepts/scaling/terms/domain/scaleOrdinal/scaleOrdinal.md:27]: Updated

File: content/d3/concepts/scaling/terms/domain/scaleOrdinal/scaleOrdinal.md
Comment (A-Kiwams) [reply content/d3/concepts/scaling/terms/domain/scaleOrdinal/scaleOrdinal.md]: Updated @dakshdeepHERE

File: content/d3/concepts/scaling/terms/domain/scaleOrdinal/scaleOrdinal.md
Comment (dakshdeepHERE) [inline content/d3/concepts/scaling/terms/domain/scaleOrdinal/scaleOrdinal.md:9]:
```suggestion
CatalogContent:
  - 'learn-d3'
  - 'paths/data-science'
```

File: content/d3/concepts/scaling/terms/domain/scaleOrdinal/scaleOrdinal.md
Comment (dakshdeepHERE) [inline content/d3/concepts/scaling/terms/domain/scaleOrdinal/scaleOrdinal.md:8]:
```suggestion
  - 'D3'
  - 'Methods'
  - 'Objects'
```

File: content/d3/concepts/scaling/terms/domain/scaleOrdinal/scaleOrdinal.md
Comment (dakshdeepHERE) [inline content/d3/concepts/scaling/terms/domain/scaleOrdinal/scaleOrdinal.md:23]:
```suggestion
scaleOrdinal(domain, range)

It has two input parameters: domain and range:

- `domain`: This specifies the scale's maximum and minimum values.
- `range`: Values in the range correspond to values in the domain.
```

File: content/d3/concepts/scaling/terms/domain/scaleOrdinal/scaleOrdinal.md
Comment (dakshdeepHERE) [inline content/d3/concepts/scaling/terms/domain/scaleOrdinal/scaleOrdinal.md:38]: what exactly is this? I don't see the above code printing anything.

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
I have few comments and a question for you @yoiang

File: content/d3/concepts/scaling/terms/domain/scaleOrdinal/scaleOrdinal.md
Comment (A-Kiwams) [reply content/d3/concepts/scaling/terms/domain/scaleOrdinal/scaleOrdinal.md]: modified with different [example](https://github.com/Codecademy/docs/pull/3993/commits/8811f8e9090c721c248f3a8e6b984bfb23cfe95c), @dakshdeepHERE

**Comment by dakshdeepHERE** [comment]
The commits you requested from you are still pending @A-Kiwams. Can you please work on them so we can move forward with the reviews.

**Comment by A-Kiwams** [comment]
> The commits you requested from you are still pending @A-Kiwams. Can you please work on them so we can move forward with the reviews.

I made a commit using a different [example](https://github.com/Codecademy/docs/pull/3993/commits/8811f8e9090c721c248f3a8e6b984bfb23cfe95c) and a second commit to change a variable [name](https://github.com/Codecademy/docs/pull/3993/commits/a1fc445e5bee630fdf38ac0ef348ea25b6f2f927) two days ago, but it has not been reviewed yet. @dakshdeepHERE

File: content/d3/concepts/scaling/terms/domain/scaleOrdinal/scaleOrdinal.md
Comment (dakshdeepHERE) [inline content/d3/concepts/scaling/terms/domain/scaleOrdinal/scaleOrdinal.md:44]: @A-Kiwams my main issue at start was this example that you've given. See the description that you have given for the snippet and then check the snippet. The description says you will map years with the invention. But the code you are trying to map the inventors with the inventions. Please fix this so we can move to the second review.

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]

**Comment by dakshdeepHERE** [comment]
@A-Kiwams I checked that change already. When a maintainer as for changes you commit them if you have doubts then ask the doubts in the comments of that commit requests itself other then that apply the commits on the other comments.

**Comment by dakshdeepHERE** [comment]
I am closing PR due to author's inactivity on the changes.

**Pull request closed without merging**
